lang_python/parsing/Parser_python.mly

```diff
@@ -629,13 +629,23 @@ excepthandler:
   (* python2: *)
   | EXCEPT test "," NAME ":" suite { ExceptHandler ($1, Some $2, Some $4, $6) }
 
-with_stmt: WITH with_inner { $2 $1 }
+with_stmt:
```

Comment:
BTW we are in the process of switching to tree-sitter for handling Python, so the tree-sitter grammar for Python
can be a good source of inspiration for those fixes:
https://github.com/tree-sitter/tree-sitter-python/blob/78c4e9b6b2f08e1be23b541ffced47b15e2972ad/grammar.js#L345

Reply:
It looks like the linked grammar might be wrong---you're allowed to have a trailing comma if you're using parens, so we might need to add that to avoid a regression if we switch to the tree-sitter parser.